markdown-preview-eww.el

```diff
@@ -42,19 +42,75 @@
   "Filename of converted html.")
 
 (defvar markdown-preview-eww-waiting-idling-second 1
-  "Seconds of convert waiting")
+  "Seconds of convert waiting.")
 
-(defun markdown-preview-eww-convert-command (output-file-name)
-  (format "require \"redcarpet\"
+(defcustom markdown-preview-eww-convert-command
+  nil
+  "Symbol of markdown convert function.")
+
+(defcustom markdown-preview-eww-major-mode-default-dialect
+  'markdown
+  "Default Markdown dialect.
+
+markdown: processor own default dialect
```

Comment:
Instead of listing the options in the docstring only, use `:tag` to show their descriptive names in the Customize interface.